app/services/symphony_client_parser.rb

```diff
@@ -2,8 +2,9 @@
 
 # Responsible for parsing SymphonyClient responses.
 class SymphonyClientParser
-  def self.parsed_response(symphony_client, symphony_call, **params)
-    client_response = symphony_client.send(symphony_call, **params)
+  def self.parsed_response(symphony_client, symphony_call, *params)
+    kwargs = params.first.nil? ? {} : params.first
```

Comment:
Should we just pass `params` instead of `params.first` ? Are there other places where we're calling `symphony_client.send` that may need refactoring, too?

Reply:
`params` gives a hash inside of an array, so first gives us just the hash that we need

We don't call `symphony_client.send` anywhere else, and it looks like the other calls for the parser were already reformatted.

Reply:
When I try with **params, 
```
def self.parsed_response(symphony_client, symphony_call, **params)
  client_response = symphony_client.send(symphony_call, **params)
  JSON.parse(client_response.body)
end
```
ILL holds working on my local. Do we need this change or I might be missing sth here?  I think the change in the bib_info method fixes it?

Reply:
Yeah I think the change in the bib_info is probably sufficient if *params is changed back to **params.  Might still have to handle nils with the **kwargs part, though.